<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="config_path" default = "$(find image_to_bag)/config/config.yaml" />
  <node pkg="image_to_bag" name="image_to_bag_node" type="image_to_bag_node" output="screen">
    <param name="config_file" type="string" value="$(arg config_path)" />
  </node>
</launch>